system-addon/content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -228,25 +124,64 @@ export class _CollapsibleSection extends React.PureComponent {
     return <span className={`icon icon-small-spacer icon-${icon || "webextension"}`} />;
   }
 
+  onMenuButtonClick(event) {
+    event.preventDefault();
+    this.setState({showContextMenu: true});
+  }
+
+  onMenuButtonMouseEnter() {
+    this.setState({menuButtonHover: true});
+  }
+
+  onMenuButtonMouseLeave() {
+    this.setState({menuButtonHover: false});
+  }
+
+  onMenuUpdate(showContextMenu) {
+    this.setState({showContextMenu});
+  }
+
   render() {
     const isCollapsible = this.props.prefName in this.props.Prefs.values;
     const isCollapsed = getCollapsed(this.props);
-    const {enableAnimation, isAnimating, maxHeight} = this.state;
-    const {id, infoOption, eventSource, disclaimer} = this.props;
+    const {enableAnimation, isAnimating, maxHeight, menuButtonHover, showContextMenu} = this.state;
+    const {id, eventSource, disclaimer, title, menuOptions, name, prefName, showPrefName, privacyNoticeURL, dispatch} = this.props;
     const disclaimerPref = `section.${id}.showDisclaimer`;
     const needsDisclaimer = disclaimer && this.props.Prefs.values[disclaimerPref];
+    const active = menuButtonHover || showContextMenu;
 
     return (
-      <section className={`collapsible-section ${this.props.className}${enableAnimation ? " animation-enabled" : ""}${isCollapsed ? " collapsed" : ""}`}>
+      <section className={`collapsible-section ${this.props.className}${enableAnimation ? " animation-enabled" : ""}${isCollapsed ? " collapsed" : ""}${active ? " active" : ""}`}>
         <div className="section-top-bar">
           <h3 className="section-title">
             <span className="click-target" onClick={isCollapsible && this.onHeaderClick}>
               {this.renderIcon()}
-              {this.props.title}
+              {title}
             {isCollapsible && <span className={`collapsible-arrow icon ${isCollapsed ? "icon-arrowhead-forward" : "icon-arrowhead-down"}`} />}
             </span>
           </h3>
-          {infoOption && <InfoIntl infoOption={infoOption} dispatch={this.props.dispatch} />}
+          <div>
+            <button
+              className="context-menu-button icon"
+              onClick={this.onMenuButtonClick}
+              onMouseEnter={this.onMenuButtonMouseEnter}
+              onMouseLeave={this.onMenuButtonMouseLeave}>
+              <span className="sr-only">
+                <FormattedMessage id="section_context_menu_button_sr" />
+              </span>
+            </button>
+            <SectionMenu
+              options={menuOptions}
+              name={name}
```

Comment:
`name` is unused here too.

Reply:
Oh odd. This is indeed outdated. I wonder what view github was showing me earlier…